src/EPR.Calculator.API/Controllers/CalculatorController.cs

```diff
@@ -96,7 +97,7 @@ public async Task<IActionResult> Create([FromBody] CreateCalculatorRunDto reques
                         Financial_Year = request.FinancialYear,
                         CreatedBy = request.CreatedBy,
                         CreatedAt = DateTime.Now,
-                        CalculatorRunClassificationId = 1,
+                        CalculatorRunClassificationId = (int)RunClassification.Running,
```

Comment:
Might change based on what you decide for my above comment.

Reply:
made it consistent